Package.swift

```diff
@@ -0,0 +1,30 @@
+// swift-tools-version: 5.6
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Backtrace",
+    platforms: [
+        .iOS(.v10),
+        .macOS(.v10_10),
+        .tvOS(.v12)
+    ],
+    products: [
+        .library(
+            name: "Backtrace",
+            targets: ["Backtrace"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/microsoft/plcrashreporter.git", from: "1.10.0"),
```

Comment:
Is it possible to create an initial version of this that depends on our [PLCR fork](https://github.com/backtrace-labs/plcrashreporter)? Probably not, since our fork is not SwiftPM enabled, right?

Reply:
Yes its not possible as its not enabled with SwiftPM and out of date as well.